marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/WriteDataTest.java

```diff
@@ -48,6 +47,25 @@ void ingestThreeFruitsWithBatchSizeOfTwo() throws IOException {
         verifyFruitCount(3, "The commit call should result in the 3rd fruit being ingested");
     }
 
+
+    @Test
+    public void testBulkIngestWithoutUriPrefix() throws IOException {
+        DataWriter<InternalRow> dataWriter = buildDataWriter("1", "");
+        dataWriter.write(buildRow("pineapple", "green"));
+
+        String uriQuery = "cts.uris('', null, cts.andQuery([\n" +
+            "  cts.jsonPropertyValueQuery('fruitName', 'pineapple')\n" +
+            "]))";
+
+        EvalResultIterator uriQueryResult = getHubClient().getStagingClient().newServerEval().javascript(uriQuery).eval();
```

Comment:
No need to change here, but most of the time, it's sufficient to just do this:

    String uri = getHubClient().getStagingClient().newServerEval().javascript(query).evalAs(String.class);

If that fails because there's not a URI, that's totally fine. And we don't need to worry about verifying that there's not another URI - we know the database was cleaned before the test ran, so it's safe to assume there's one or zero URIs.